rest-api/tools/import_organizations.py

```diff
@@ -124,7 +124,8 @@ def __init__(self):
 
   def _entity_from_row(self, row):
     google_group = row[SITE_SITE_ID_COLUMN].lower()
-    organization = self.organization_dao.get_by_external_id(row[SITE_ORGANIZATION_ID_COLUMN])
+    organization = self.organization_dao.get_by_external_id(
+                                        row[SITE_ORGANIZATION_ID_COLUMN].upper())
```

Comment:
You can't just change it in these places. You have to call upper() everywhere that awardee ID and organization ID are used.

https://github.com/all-of-us/raw-data-repository/blob/master/rest-api/tools/import_organizations.py#L80
https://github.com/all-of-us/raw-data-repository/blob/master/rest-api/tools/import_organizations.py#L102

etc.

Reply:
think I got it